apps/web/components/MailingListSignup/MailingListSignup.tsx

```diff
@@ -80,9 +65,19 @@ export const MailingListSignup: React.FC<MailingListSignupProps> = ({
               type: 'success',
               message: successMessage,
             })
+          } else {
```

Comment:
Is there no case for "Already signed up"?

Reply:
Mailchimp is not configured to return "already signed up", it always just returns "success". So I decided to omit this.